CHANGELOG.md

```diff
@@ -2,9 +2,18 @@
 
 Converts VCF variants into a [FHIR Genomics Diagnostic Report](http://hl7.org/fhir/uv/genomics-reporting/index.html). 
 
-## Additions
+## Added

```

Comment:
This change was part of previous commit :) . You can read more about interactive [rebase](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase). It makes it easier to modify a commit not at the top.  
For this case, You can do :
```
stage your change and instead of including to the top commit you can commit as below.
git commit --fixup <commit-id for which this change belongs too?
// Do interactive rebase to merge top 3 commits.
git rebase -i HEAD~3 --autosquash
```

Reply:
This is still not fixed. Can you move this change to other commit.